e107_plugins/login_menu/login_menu_class.php

```diff
@@ -117,7 +117,7 @@ function get_external_list($sort = true)
     function parse_external_list($active=false, $order=true)
 	{
         //prevent more than 1 call
-        if(($tmp = e107::getRegistry('loginbox_elist')) !== FALSE) return $tmp;
+        if(($tmp = e107::getRegistry('loginbox_elist', false)) !== null) return $tmp;
```

Comment:
Don't add the second argument (`, false`).  Otherwise, the comparison to `null` will fail because the `false` value will be passed to the comparison.

Reply:
My mistake... next attempt.